lib/trento_web/openapi/v1/schema/platform.ex

```diff
@@ -25,6 +25,54 @@ defmodule TrentoWeb.OpenApi.V1.Schema.Platform do
     })
   end
 
+  defmodule ApiKeySettings do
+    @moduledoc false
+    OpenApiSpex.schema(%{
+      title: "ApiKeySettings",
+      description: "Settings for Api Key generation",
+      type: :object,
+      additionalProperties: false,
+      properties: %{
+        api_key_created_at: %Schema{
```

Comment:
I would rename some field names:
`api_key_created_at` -> `created_at`
`generated_api_key` -> `api_key`
`api_key_expire_at` -> `expire_at`

At the end, we know that we are talking about the api key

Reply:
Yes, on the database we will have the name with prefix, for both tables and we alias them in the ecto schema